Node.js 15 issue

```
➜ npx clean-publish
/usr/bin/env: ‘node\r’: No such file or directory
npm ERR! code 127
npm ERR! path /home/ai/Dev/storeon
npm ERR! command failed
npm ERR! command sh -c clean-publish

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ai/.npm/_logs/2020-10-22T23_37_49_537Z-debug.log
```

@shashkovdanil maybe we use a Windows `\r\n` and it causes a problem in npm 7 on Linux?

--- a/.editorconfig
+++ b/.editorconfig
@@ -1,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true

